apis/shared.go

```diff
@@ -4,19 +4,20 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
-	"github.com/go-playground/validator/v10"
 	"net/http"
-	ctrl "sigs.k8s.io/controller-runtime"
 	"strings"
 
-	"code.cloudfoundry.org/cf-k8s-api/presenters"
+	"github.com/go-playground/validator/v10"
+	ctrl "sigs.k8s.io/controller-runtime"
+
+	"code.cloudfoundry.org/cf-k8s-api/presenter"
 )
 
 var Logger = ctrl.Log.WithName("Shared Handler Functions")
 
 type requestMalformedError struct {
 	httpStatus    int
-	errorResponse presenters.ErrorsResponse
+	errorResponse presenter.ErrorsResponse
```

Comment:
Is `presenter.ErrorsResponse` correct now that we've singularized a bunch of stuff?